packages/edit-site/src/plugins/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import CopyContentMenuItem from './copy-content-menu-item';
+import ManageBlocksMenuItem from './manage-blocks-menu-item';
+import KeyboardShortcutsHelpMenuItem from './keyboard-shortcuts-help-menu-item';
+import ToolsMoreMenuGroup from '../components/header/tools-more-menu-group';
+import WelcomeGuideMenuItem from './welcome-guide-menu-item';
+
+registerPlugin( 'edit-site', {
+	render() {
+		return (
+			<>
+				<ToolsMoreMenuGroup>
+					{ ( { onClose } ) => (
+						<>
+							<ManageBlocksMenuItem onSelect={ onClose } />
+							<MenuItem
+								role="menuitem"
+								href={ addQueryArgs( 'edit.php', {
+									post_type: 'wp_block',
+								} ) }
+							>
+								{ __( 'Manage all reusable blocks' ) }
+							</MenuItem>
+							<KeyboardShortcutsHelpMenuItem
+								onSelect={ onClose }
+							/>
+							<WelcomeGuideMenuItem />
+							<CopyContentMenuItem />
+							<MenuItem
+								role="menuitem"
+								href={ __(
+									'https://wordpress.org/support/article/wordpress-editor/'
+								) }
+								target="_blank"
+								rel="noopener noreferrer"
+							>
+								{ __( 'Help' ) }
+							</MenuItem>
+						</>
+					) }
+				</ToolsMoreMenuGroup>
```

Comment:
cc @youknowriad @epiqueras since this file (and most of the sub-directories here) are going to be almost identical between edit-sit and edit-post, where do you think would be a good place to share this code? Should we copy this to `interface`? or maybe `editor`? Not really up to date on our latest thoughts for increasing reusability here.

Reply:
>The Interface Package contains the basis the start a new WordPress screen as Edit Post or Edit Site. The package offers a data store and a set of components. The store is useful to contain common data required by a screen (e.g., active areas). The information is persisted across screen reloads. The components allow one to implement functionality like a sidebar or menu items. Third-party plugins by default, can extend them.

I guess it makes sense in there.

Reply:
What if we also put feature toggle state there? (e.g. full screen, focus mode, block toolbar)

Reply:
What do you think @youknowriad?

Reply:
I don't think it makes sense for "Interface" as "interface" is not editor specific and AFAIK, these tools are?

I've been thinking about this and maybe this will change later but right now, the edit-site package is the exact same thing as edit-post loading the template as top level with some additional UI on the header. 

I do wonder whether it's simpler to just use edit-post with extensibility in the end, but, at the moment it's not clear how far we're going to diverge. So since we already have a good chunk of the screen done, I think we should continue as is,  duplicating code if needed, trying to share smaller components (if we find the right fit for them, sometimes there isn't). And once we'll have most things done, we could try to reanswer that question again. (This is me saying, it's fine to duplicate things over creating early abstractions unless the abstraction is clear and semantically correct)

Reply:
Yeah, I'm also more confident in delaying this decision, because it's too early to make the right one.

Reply:
> I think we should continue as is, duplicating code if needed, trying to share smaller component

Yeah, I don't disagree that it's hard to come up with a good abstraction right now. I feel so bad duplicating this many modals. 🤷

Reply:
I dug into this more, and I really don't want to put the work in to duplicating this code into another package, so I may put this on hold for the time being. I know for certain that we do not want this to be duplicated long-term because it's such a broad set of code and features.

This PR would probably be some thousands of LOC and introduce dozens of new files to the site editor. I don't think it's worth duplicating at this time being because:
- in this early development/alpha stage, I don't see these features being all that useful.
- we'll have to put in the effort to un-duplicate this and put it into a semantically correct place in the future.

Why go through that much work twice if the tools aren't very useful for us at the time being?

for small things like the focus mode or the block toolbar, duplication is fine because the real source of the logic is in the block-editor package, so it can be reused.

Reply:
This leads me to: how much of the "basic UI parity" features should we really duplicate before we come up with a good abstraction. I don't think "all of them" makes sense because there would be so much to basically just copy over and hook in. And then we have two copies of everything floating around and we aren't any closer to having a better abstraction. :/

Reply:
As much as possible.

Reply:
moving this conversation to #21245